gst/rtp/gstrtph264depay.c

```diff
@@ -1260,3 +1333,26 @@ gst_rtp_h264_depay_plugin_init (GstPlugin * plugin)
   return gst_element_register (plugin, "rtph264depay",
       GST_RANK_SECONDARY, GST_TYPE_RTP_H264_DEPAY);
 }
+
+static gboolean
+gst_rtp_h264_depay_packet_lost (GstRTPBaseDepayload * depay, GstEvent * event)
+{
+  GstRtpH264Depay *rtph264depay = GST_RTP_H264_DEPAY (depay);
+  GstClockTime ts = 0;
+  guint seq = 0;
+
+  const GstStructure *structure = gst_event_get_structure (event);
+  gst_structure_get_uint (structure, "seqnum", &seq);
+  gst_structure_get_uint64 (structure, "timestamp", &ts);
+
+  rtph264depay->lost_seq = seq;
+  rtph264depay->lost_ts = ts;
+
+  gst_pad_push_event (GST_RTP_BASE_DEPAYLOAD_SINKPAD (depay),
+      gst_event_new_custom (GST_EVENT_CUSTOM_UPSTREAM,
+          gst_structure_new_empty ("GstForceKeyUnit")));
+
+  GST_RTP_BASE_DEPAYLOAD_CLASS (parent_class)->packet_lost (depay, event);
+
+  return TRUE;
```

Comment:
Please consider doing it like in handle_event function:
```
return
      GST_RTP_BASE_DEPAYLOAD_CLASS (parent_class)->handle_event (depay, event);
```

Reply:
forgot )))